bin/mocha-server.js

```diff
@@ -37,31 +36,21 @@ function getTestPaths(dir) {
 
 const testPaths = getTestPaths(path.join(__dirname, "../src"));
 
-projectConfig.entry.bundle = projectConfig.entry.bundle.concat(testPaths);
```

Comment:
I'll pull this work into a new PR. It fixes the mocha web runner which is great though